src/ray/common/runtime_env_manager.cc

```diff
@@ -21,9 +21,6 @@ void RuntimeEnvManager::AddURIReference(const std::string &hex_id,
                                         const rpc::RuntimeEnv &runtime_env) {
   const auto &uris = runtime_env.uris();
   for (const auto &uri : uris) {
-    if (unused_uris_.count(uri)) {
-      unused_uris_.erase(uri);
-    }
```

Comment:
what's the context for this change?

Reply:
As far as I can tell it wasn't doing anything, it was just getting populated and deleted but was never read anywhere else.  It might have been leftover from some earlier implementation

Reply:
gotcha